proto/message.py

```diff
@@ -388,6 +393,7 @@ def to_dict(
             preserving_proto_field_name (Optional(bool)): An option that
                 determines whether field name representations preserve
                 proto case (snake_case) or use lowerCamelCase. Default is True.
+            including_default_value_fields (Optional(bool)): An option that determines whether the default field values should be included in the results. Default is True.
```

Comment:
Please wrap here as well:
```suggestion
            including_default_value_fields (Optional(bool)): An option that
            determines whether the default field values should be included in
            the results. Default is True.
```